### PR TITLE
feat: Support for configuring authentication OTP policy (#394)

### LINKED ISSUE #394: Support for configuring authentication OTP policy
Hi, 

i have not come across means to change the "OTP Policy" in the authentication section of keycloak. 
Default provided is SHA1 but would like to configure SHA256 for my terraform provided realms.

### PR DESCRIPTION
**Pull request by nolte**
Adding support for configuring authentication OTP policy of some Realm, will be fixed #394

### CODE REVIEW

File: provider/resource_keycloak_realm.go [hunk 1]
```diff
@@ -7,6 +7,56 @@ import (
...
+		},
+		"digits": {
+			Type: schema.TypeInt,
+			Elem: &schema.Schema{
+				Type: schema.TypeInt,
+			},
+			Default:  6,
+			Optional: true,
+		},
+		"initial_counter": {
```
Comment (mrparkers) [inline provider/resource_keycloak_realm.go:34]: I don't actually see the setting for this in the latest three Keycloak versions.  Am I missing something?

**Review by mrparkers — COMMENTED** [review]
Thanks for the PR, overall this LGTM, just had one question

File: provider/resource_keycloak_realm.go (hunk 1 above)
Comment (nolte) [reply provider/resource_keycloak_realm.go:34]: The Option is only visible if you use a  `OTP Type:` _Counter Based_  , for the Default Type, _Time Based_, you will got the field `OTP Token Period` ( [period](https://github.com/mrparkers/terraform-provider-keycloak/blob/efa42c17c5b5e549861dfc5a6983b2f4672ed868/provider/resource_keycloak_realm.go#L50)).

Comment (mrparkers) [reply provider/resource_keycloak_realm.go:34]: Perfect, I just missed that then.  Thanks for your help

**Review by mrparkers — APPROVED** [review]
LGTM 🚀

**Pull request merged**